hydra-module.nix

```diff
@@ -1,45 +1,113 @@
-{ config, pkgs, lib ? pkgs.lib, ... }:
+{ config, pkgs, lib, ... }:
 
-with lib;
+# ------------------------------------------------------------------------------
+# FIXME: send this stuff upstream to nixpkgs
+# vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
 
-let
+with rec {
+  inherit (lib) mkIf mkOption;
 
+  forceWHNF = x: builtins.seq x x;
+  forceDeep = x: builtins.deepSeq x x;
```

Comment:
This pull request does not use `forceWHNF` and `forceDeep`

Also `forceWHNF` does not do anything because `seq x x = x`

Reply:
Good point on `forceWHNF`.